tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -94,9 +95,11 @@ Get RHODS version
     #@{list} =  OpenShiftCLI.Get  kind=ClusterServiceVersion  label_selector=olm.copiedFrom=redhat-ods-operator
```

Comment:
Could you add a force-reload argument to force getting the info from oc even if the global variable is already set?